lib/tasks/tiny_mce_asset.rake

```diff
@@ -6,4 +6,75 @@ namespace :tiny_mce_asset do
                        'result_text_id IS ? AND created_at < ?',
                        nil, nil, 7.days.ago)
```

Comment:
The Rake task `remove_obsolete_images` doesn't do anything. It's just a `where` statement. I assume a `.destroy_all` is missing at the end of the query.

Reply:
Shouldn't be here OR instead of AND?